src/tracks/playabletrack/wavetrack/ui/WaveClipTrimHandle.cpp

```diff
@@ -118,18 +178,40 @@ class WaveClipTrimHandle::AdjustBorder final : public WaveClipTrimHandle::ClipTr
       return false;
    }
```

Comment:
A general comment: I'm reviewing changes in the class AdjustBorders.  The other subclass of ClipTrimPolicy is AdjustBetweenBorders.  That class is not constructed anywhere.  Is there a mistake?

Reply:
It's not a mistake, AdjustBetweenBorders moves borders of both clips if they are close enough, and it was my understanding of how things should happen, and turned out that I was wrong. But I was told that we may need this in the future.